### PR TITLE
With routing

### PR DESCRIPTION
**Pull request by zawar-mughal**

### CODE REVIEW

File: remote/webpack.config.js [hunk 1]
```diff
@@ -51,10 +51,17 @@ module.exports = (_, argv) => ({
         react: {
           singleton: true,
           requiredVersion: deps.react,
+          // eager: true,
         },
+        // "react-router-dom": {
+        //   singleton: true,
+        //   requiredVersion: deps["react-router-dom"],
+        //   eager: true,
+        // },
```
Comment (buildatlas-dev) [inline remote/webpack.config.js]: Please remove this.

File: remote/src/components/NavBar.jsx [hunk 2]
```diff
@@ -3,9 +3,9 @@ import { Link, Outlet } from "react-router-dom";
 function NavBar() {
```
Comment (buildatlas-dev) [inline remote/src/components/NavBar.jsx]: The name of this component is misleading. can we please change it?

File: remote/src/components/Routing.jsx [hunk 3]
```diff
@@ -1,25 +1,43 @@
...
-          <Route path="/remote-home" element={<Home />} />
-          <Route path="/remote-contact" element={<Contact />} />
-          <Route path="/remote-about" element={<About />} />
-        </Routes>
-      </BrowserRouter>
+      <h1>Remote...</h1> <br />
+      <br />
+      <NavBar />
+      <Routes>
+        <Route path="/*" element={<PageNotFound />} />
```
Comment (buildatlas-dev) [inline remote/src/components/Routing.jsx]: Try using only * I believe that could also work fine.

File: shell/src/components/ShellRouting.jsx [hunk 4]
```diff
@@ -16,10 +18,16 @@ export default function ShellRouting() {
           <Route path="/" element={<Home />} />
           <Route path="/home" element={<Home />} />
           <Route path="/about" element={<About />} />
-          {/* <Route path="/remote/*" element={<Remote/>} /> */}
```
Comment (buildatlas-dev) [inline shell/src/components/ShellRouting.jsx:19]: Remove this.

File: shell/webpack.config.js [hunk 5]
```diff
@@ -52,14 +52,16 @@ module.exports = (_, argv) => ({
           singleton: true,
           requiredVersion: deps.react,
         },
-        "react-router-dom": {
-          singleton: true,
-          requiredVersion: deps["react-router-dom"],
-        },
-        // "react-dom": {
```
Comment (buildatlas-dev) [inline shell/webpack.config.js:59]: Remove this

File: remote/webpack.config.js (hunk 1 above)
Comment (zawar-mughal) [reply remote/webpack.config.js]: Done.

File: remote/src/components/NavBar.jsx (hunk 2 above)
Comment (zawar-mughal) [reply remote/src/components/NavBar.jsx]: Done

File: remote/src/components/Routing.jsx (hunk 3 above)
Comment (zawar-mughal) [reply remote/src/components/Routing.jsx]: Done

File: shell/src/components/ShellRouting.jsx (hunk 4 above)
Comment (zawar-mughal) [reply shell/src/components/ShellRouting.jsx:19]: Done

File: shell/webpack.config.js (hunk 5 above)
Comment (zawar-mughal) [reply shell/webpack.config.js:59]: Done

**Review by buildatlas-dev — APPROVED** [review]

**Pull request merged**